dbscripts/schema.sql

```diff
@@ -0,0 +1,35 @@
+-- Table: SP_APP.
```

Comment:
It'd be better to add meaningful comments instead of repeating the table name

Reply:
Addressed with 7fb47d4c0a952257710725d482027ce293e9bef8